2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+sustainability funding a software and the necessary reevaluation to
+monitor the state of the software over time.
+
+Selection Criteria
+------------------
+
+Several evaluation schemes for research software have been proposed
+before and led to the formulation of first recommendations (Katerbow and
+Feulner 2018; Scheliga et al. 2019). Gomez-Diaz & Recio suggested the
+CDUR based on Citation, Dissemination (including aspects like license,
+web site, contact point), Use, and Research (output) (Gomez-Diaz and
+Recio 2019). Lamprecht et al. rephrased the FAIR data
+principles (Wilkinson et al. 2016) for research software (Lamprecht et
+al. 2019). Hasselbring et al. found that the adoption of FAIR principles
+is different between fields with an emphasis on reuse in computer
+science as opposed to a reproducibility focus in computational
+science (Hasselbring et al. 2019). Fehr et al. collected a set of best
+practices for the setup and publication of numerical experiments (Fehr
+et al. 2016). Jiménez et al. boiled it down to four best
+practices (Jiménez et al. 2017): public source code, community registry,
+license, and governance. Hsu et al. (Hsu, Hutchison, and Langseth 2019)
+proposed a framework of seven sustainability influences (outputs
+modified, code repository used, champion present, workforce stability,
+support from other organizations, collaboration/partnership, and
+integration with policy). They found that the various outputs are widely
+accessible, but not necessarily sustained or maintained. Projects with
+most sustainability influences often became institutionalized and met
+required needs of the community (Hsu, Hutchison, and Langseth 2019). In
+the field of open source software, the CHAOSS (Community Health
+Analytics Open Source Software) project has developed metrics to
+evaluate sustainability (CHAOSS 2019). One objective of CHAOSS is to
+automatically generate project health reports based on software that
+evaluates the metrics, with most of the metrics already covered. The UK
+Software Sustainability Institute (SSI) suggested both a subjective
+tutorial-based and a more objective criteria-based software evaluation
+scheme (Jackson, Crouch, and Baxter 2019), the latter being available as
+an online form (SSI 2019a).
+
+We drew inspiration from all these works and suggest a set of criteria
+to base reviews for sustainable funding on. This set comprises
+mandatory, hard criteria that we think have to be fulfilled across
+domains (highlighted in bold) and additional desirable, soft criteria
+that can be implemented to different degrees depending on the use case
+and domain-specific software development requirements. The soft criteria
+should be evaluated in a structured way by the reviewers with a specific
+response for each section rather than one running text. The fact that
+most of these criteria will be considered in any software management
+plan (SMP, compare section   on p. ) (SSI 2019b) highlights its
```

Comment:
```suggestion
plan (SMP, compare section ???  on p. ???, SSI 2019b) highlights its
```

Reply:
worked in LaTeX, doesn't work in Markdown